packages/core/src/components/workflow-steps/workflow-steps.tsx

```diff
@@ -81,8 +81,11 @@ export class WorkflowSteps {
         'selected',
         this.selectedIndex === index ? 'true' : 'false'
       );
-      if (index === 0) element.setAttribute('position', 'first');
-      if (index === steps.length - 1) element.setAttribute('position', 'last');
+      if (index === 0 && steps.length > 1)
+        element.setAttribute('position', 'first');
```

Comment:
Why do we use `element.setAttribute` to define all the attributes on the step? Why not just simply e.g. `element.position = 'first'`. Because element should be type of `HTMLIxWorkflowStep`